clubs/serializers.py

```diff
@@ -170,23 +178,50 @@ def get_image_url(self, obj):
         else:
             return self.context['request'].build_absolute_uri(obj.image.url)
 
+    def get_parent_orgs(self, obj):
+        return []
+        # return [c.slug for c in obj.parent_orgs.all()]
+
     def create(self, validated_data):
         """
         Manual create method because DRF does not support writable nested fields by default.
         """
         # assign tags to new club
-        has_tags = 'tags' in validated_data
+        tags = None
+        parent_orgs = None
+        badges = None
 
-        if has_tags:
+        if 'tags' in validated_data:
             tags = []
             for tag in validated_data.pop('tags'):
                 tags.append(Tag.objects.get(**tag))
 
+        # Don't let clubs add parent orgs for now; they will be added through the admin
```

Comment:
I agree we probably shouldn't let clubs define a parent organization. We should probably delete the commented code then.

Reply:
But parent orgs should be able to define relationships that include them. And that would happen through this serializer, if you're logged in with access for a parent club. Think it's worth keeping this in for now.